onyo/onyo_arguments.py

```diff
@@ -10,36 +10,49 @@
         required=False,
         default=Path.cwd(),
         type=directory,
-        help='Run Onyo commands from inside of DIR'),
+        help="""
+            Run as if Onyo is started in DIR instead of the current working
+            directory.
+        """
+    ),
 
     'debug': dict(
         args=('-d', '--debug'),
         required=False,
         default=False,
         action='store_true',
-        help='Enable debug logging'),
+        help="""
+            Enable debug logging.
+        """
+    ),
 
     'version': dict(
         args=('-v', '--version'),
         action='version',
         version='%(prog)s {version}'.format(version=__version__),
-        help="Print onyo's version and exit"),
+        help="""
+            Print Onyo's version and exit.
+        """
+    ),
 
     'quiet': dict(
         args=('-q', '--quiet'),
         required=False,
         default=False,
         action='store_true',
-        help=(
-            'Silence messages printed to stdout. Does not suppress interactive '
-            'editors. Requires the --yes flag')),
+        help="""
+            Silence messages printed to stdout; does not suppress interactive
+            editors. Requires the ``--yes`` flag.
+        """
+    ),
 
     'yes': dict(
         args=('-y', '--yes'),
         required=False,
         default=False,
         action='store_true',
-        help=(
-            'Respond "yes" to any prompts. The --yes flag is required to use '
-            '--quiet')),
+        help="""
+            Respond "yes" to any prompts.
```

Comment:
Now I am definitely out of my league but I find "any prompt" or "all prompts" sounds more correct?

Reply:
Both are ok. IMO "any" means that there could or could not be prompts, and we will answer yes to them. While "all" implies that there /will/ be prompts, and we will answer yes to them.

If you find this to be confusing though (as opposed to a nitpick), then I'm comfortable changing it to improve likelihood of comprehension.

Let me know what you think. :-)

Reply:
Hm yeah, the "all" argument I see. "Any prompt" in singular still sounds the smoothest for me :)

Reply:
The noun following "any" can be either singular or plural, but is usually plural (in my experience).

"We will say hello to any people we see."

I will leave this as is. Thanks for responding. :-)